libafl/Cargo.toml

```diff
@@ -46,6 +46,9 @@ llmp_compression = ["gzip"] # llmp compression using GZip
 llmp_debug = [] # Enables debug output for LLMP
 llmp_small_maps = [] # reduces initial map size for llmp
 llmp_broker_timeouts = ["std"] # The broker loop will yield occasionally, even without status messages from client nodes
+# Manager features
+count_process_execution = []
+no_count_newtestcases = []
```

Comment:
features shouldn't be negative according to clippy

Reply:
why not use `count_newtestcases` and invert the branches?

Reply:
they will never land main